packages/angular/src/generators/stories/lib/projects.ts

```diff
@@ -0,0 +1,32 @@
+import type {
+  NxJsonProjectConfiguration,
+  ProjectConfiguration,
+  Tree,
+} from '@nrwl/devkit';
+import { joinPathFragments, readProjectConfiguration } from '@nrwl/devkit';
+
+export function getE2EProject(
+  tree: Tree,
+  e2eProjectName: string
+): ProjectConfiguration & NxJsonProjectConfiguration {
+  let e2eProject: ProjectConfiguration & NxJsonProjectConfiguration;
+  try {
+    e2eProject = readProjectConfiguration(tree, e2eProjectName);
+  } catch {
+    e2eProject = undefined;
+  }
+
+  return e2eProject;
+}
+
+export function getProjectPath(tree: Tree, projectName: string): string {
```

Comment:
any reason we copied this into the Angular stories folder? I think initially there was something similar on the `@nrwl/workspace` level

Reply:
You're right, it's in the `@nrwl/workspace/src/utils/project-type`. I'll make it available in `@nrwl/workspace/src/utilities/project-type` instead and consume it from there.